File: api/src/main/java/org/egglog/api/calendargroup/model/dto/response/CalendarGroupResponse.java
package org.egglog.api.calendargroup.model.dto.response;

import lombok.*;

@Builder
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class CalendarGroupResponse {
    private Long calendarGroupId;
    private String url;
    private String alias;
}

File: client/data/src/main/java/com/org/egglog/data/setting/model/CalendarGroupResponse.kt
package com.org.egglog.data.setting.model

import androidx.room.Entity
import androidx.room.PrimaryKey
import com.org.egglog.domain.setting.model.CalendarGroup
import kotlinx.serialization.Serializable

@Entity
@Serializable
data class CalendarGroupResponse(
    @PrimaryKey val calendarGroupId: Long,
    val url: String?,
    val alias: String
)

fun CalendarGroupResponse.toDomainModel(): CalendarGroup {
    return CalendarGroup(
        calendarGroupId = calendarGroupId,
        url = url,
        alias = alias
    )
}